packages/patternfly-4/content/design-guidelines/usage-and-behavior/wizard/design.md

```diff
@@ -0,0 +1,113 @@
+---
+path: "/design-guidelines/usage-and-behavior/wizard"
+---
+# Wizard
+Wizards provide a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome. The wizard should be a familiar interface element for most modern technology users, although it does suggest a throwback to earlier, simpler days of PC computing. Still, step-by-step, directed experiences can be helpful in many cases that range from the simple to the complex. Simple object creation when certain values must be provided to software installation with many choices and branching are suitable for a wizard.
```

Comment:
I don't think we need anything after the first sentence here.

Reply:
Yeah, this was a carry-over but after more consideration I agree it can come out.